src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -666,7 +691,7 @@ public function blacklist($superGlobalName, $key) {
      * @param $superGlobalName string the name of the superglobal array, e.g. '_GET'
      * @return array $values without sensitive data
      */
-    private function masked(array $superGlobal, $superGlobalName) {
+    private function getMasked(array $superGlobal, $superGlobalName) {
```

Comment:
Why is this rename bundled with this commit? Note that the method is still private. Did you mean to make it protected as well?